### PR TITLE
br: support imdsv2 to retrive IAM Role

### PR DESCRIPTION
**Pull request by 3pointer**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [x] v8.4 (TiDB 8.4 versions)
- [ ] v8.3 (TiDB 8.3 versions)
- [ ] v8.2 (TiDB 8.2 versions)
- [ ] v8.1 (TiDB 8.1 versions)
- [ ] v7.5 (TiDB 7.5 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)

### What is the related PR or file link(s)?

- This PR is translated from:https://github.com/pingcap/docs-cn/pull/18640
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Comment by 3pointer** [comment]
translate from https://github.com/pingcap/docs-cn/pull/18640

File: br/backup-and-restore-storages.md [hunk 1]
```diff
@@ -112,6 +112,10 @@ Before backup, configure the following privileges to access the backup directory
 
 If you have not yet created a backup directory, refer to [Create a bucket](https://docs.aws.amazon.com/AmazonS3/latest/userguide/create-bucket-overview.html) to create an S3 bucket in the specified region. If necessary, you can also create a folder in the bucket by referring to [Create a folder](https://docs.aws.amazon.com/AmazonS3/latest/userguide/using-folders.html).
 
+> **Note：**
+>
+> In 2024, AWS changed the default behavior, and newly created instances now only support IMDSv2 by default. For more details, please refer to Set IMDSv2 as the default for all new instance launches in your account. Therefore, starting from v8.4.0, BR supports obtaining IAM role permissions on AWS EC2 instances with only IMDSv2 required. When using earlier versions of BR, you need to configure the instance to support both IMDSv1 and IMDSv2.
```
Comment (hfxsd) [inline br/backup-and-restore-storages.md:117]:
```suggestion
> **Note:**
>
> In 2024, AWS changed the default behavior, and newly created instances now only support IMDSv2 by default. For more details, see [Set IMDSv2 as default for all new instance launches in your account](https://aws.amazon.com/about-aws/whats-new/2024/03/set-imdsv2-default-new-instance-launches/). Therefore, starting from v8.4.0, BR supports obtaining IAM role permissions on AWS EC2 instances with only IMDSv2 enabled. When using earlier versions of BR, you need to configure the instance to support both IMDSv1 and IMDSv2.
```

**Review by BornChanger — APPROVED** [review]

**Review by Leavrth — APPROVED** [review]

File: br/backup-and-restore-storages.md [hunk 2]
```diff
@@ -112,6 +112,10 @@ Before backup, configure the following privileges to access the backup directory
 
 If you have not yet created a backup directory, refer to [Create a bucket](https://docs.aws.amazon.com/AmazonS3/latest/userguide/create-bucket-overview.html) to create an S3 bucket in the specified region. If necessary, you can also create a folder in the bucket by referring to [Create a folder](https://docs.aws.amazon.com/AmazonS3/latest/userguide/using-folders.html).
 
+> **Note:**
+>
+> In 2024, AWS changed the default behavior, and newly created instances now only support IMDSv2 by default. For more details, see [Set IMDSv2 as default for all new instance launches in your account](https://aws.amazon.com/about-aws/whats-new/2024/03/set-imdsv2-default-new-instance-launches/). Therefore, starting from v8.4.0, BR supports obtaining IAM role permissions on AWS EC2 instances with only IMDSv2 enabled. When using earlier versions of BR, you need to configure the instance to support both IMDSv1 and IMDSv2.
```
Comment (lilin90) [inline br/backup-and-restore-storages.md:117]:
```suggestion
> In 2024, AWS changed the default behavior, and newly created instances now only support IMDSv2 by default. For more details, see [Set IMDSv2 as default for all new instance launches in your account](https://aws.amazon.com/about-aws/whats-new/2024/03/set-imdsv2-default-new-instance-launches/). Therefore, starting from v8.4.0, BR supports obtaining IAM role permissions on AWS EC2 instances with only IMDSv2 enabled. When using BR of an earlier version before v8.4.0, you need to configure the instance to support both IMDSv1 and IMDSv2.
```

File: br/backup-and-restore-storages.md [hunk 3]
```diff
@@ -112,6 +112,10 @@ Before backup, configure the following privileges to access the backup directory
 
 If you have not yet created a backup directory, refer to [Create a bucket](https://docs.aws.amazon.com/AmazonS3/latest/userguide/create-bucket-overview.html) to create an S3 bucket in the specified region. If necessary, you can also create a folder in the bucket by referring to [Create a folder](https://docs.aws.amazon.com/AmazonS3/latest/userguide/using-folders.html).
 
+> **Note:**
+>
+> In 2024, AWS changed the default behavior, and newly created instances now only support IMDSv2 by default. For more details, see [Set IMDSv2 as default for all new instance launches in your account](https://aws.amazon.com/about-aws/whats-new/2024/03/set-imdsv2-default-new-instance-launches/). Therefore, starting from v8.4.0, BR supports obtaining IAM role permissions on AWS EC2 instances with only IMDSv2 enabled. When using BR of an earlier version before v8.4.0, you need to configure the instance to support both IMDSv1 and IMDSv2.
```
Comment (lilin90) [inline br/backup-and-restore-storages.md:117]:
```suggestion
> In 2024, AWS changed the default behavior, and newly created instances now only support IMDSv2 by default. For more details, see [Set IMDSv2 as default for all new instance launches in your account](https://aws.amazon.com/about-aws/whats-new/2024/03/set-imdsv2-default-new-instance-launches/). Therefore, starting from v8.4.0, BR supports obtaining IAM role permissions on Amazon EC2 instances with only IMDSv2 enabled. When using BR of an earlier version before v8.4.0, you need to configure the instance to support both IMDSv1 and IMDSv2.
```

**Review by lilin90 — APPROVED** [review]
/lgtm

**Pull request merged**